src/Consolonia.Themes.TurboVision/Themes/TurboVisionDark/TurboVisionDarkTheme.cs

```diff
@@ -1,20 +1,14 @@
 using System;
-using Consolonia.Core.Styles;
+using Avalonia.Markup.Xaml;
+using Avalonia.Styling;
 
 namespace Consolonia.Themes.TurboVision.Themes.TurboVisionDark
 {
-    // ReSharper disable once UnusedType.Global
-    public class TurboVisionDarkTheme : ResourceIncludeBase
+    public class TurboVisionDarkTheme : Styles
     {
-        public TurboVisionDarkTheme(Uri baseUri) : base(baseUri)
+        public TurboVisionDarkTheme(IServiceProvider sp = null)
         {
+            AvaloniaXamlLoader.Load(sp, this);
```

Comment:
ooooh. I didn't know there was a IServiceProvider signature for this. That's cool.

Reply:
I copypasted most of it from avalonia. Im even not sure how everything is supposed to work. I'm lost between styles, themes, and resource dictionaries.